### DISCUSSION TITLE
1.6.0

### DISCUSSION THREAD

**Discussion by alex19EP** [RHVoice/RHVoice#424] [General]
* Albanian language 
* Voices: add tatiana (#386)
* Voices: add Mikhail 
* Voices: add Marianna (#400)
* Voices: add lyubov (#412)
* Voices: add Vitaliy
* Voices: add Albanian voice, Hana, North Macedonian accent 
* Update emoji data to version 40 
* Android: Clean up and update dependencies. 
* Android: fix the version display if there are no voices installed. 
* Android: users can now select a dictionary file on their device and import it into the app. 
* Android: don't create an initial version of the config file. 
* Android: the option to import a config file. 
* Android: target Android 11. 
* Android: fix a crash. 
* Android: correct some translations. 
* Core: a language developer may set custom flags on segments in post-g2p and use them in their hts labels. 
* Core: g2p.case setting. 
* Core: word_break feature. 
* Core: more hts features. 
* Core: data-only languages. 
* Core: data-only languages get their list of graphemes from a file. 
* Core: voices may provide equalization filters.
  * Use a cascade of biquad filters.
  * Save the coefficients in 24000/eq.txt.
  * First line is a header with the version number. In case I decide to change the format.
  * Then coefficients separated by whitespace.
  * For each filter: b0, b1, b2, a0, a1, a2.
* Core: enable the limitter if equalization is used by the voice. 
* Core: general support for a simple "accented" decision tree. 
* Core: Failing to load the eq shouldn't cause the voice loading to fail. 
* Core: support comments in eq.txt. 
* Core: if the name of a phone starts with /, remove the / after syllabification. 
* Thvoice-transcribe-sentences: the -v option now also prints out gpos if not content. 
* Brazilian Portuguese: fix the pronunciation of the word backup. 
* Brazilian Portuguese: pronunciation of the ordinal indicator characters. 
* update Ukrainian dictionary 
* honor 'SOURCE_DATE_EPOCH' environment variable if it's set.
  this helps rhvoice build reproducibility.

<hr /><em>This discussion was created from the release <a href='https://github.com/RHVoice/RHVoice/releases/tag/untagged-fb16a7405f4e89a6c21d'>1.6.0</a>.</em>

**Comment by CyberTailor** [comment]
What's the public PGP key used for signing?

**Reply by alex19EP** [reply]
> What's the public PGP key used for signing?

it's my key. it can be found on [github](https://github.com/alex19ep.gpg), [keybase](https://keybase.io/alex19ep/pgp_keys.asc?fingerprint=6c7f7f22e0152a6fd5728592dad6f3056c897266), ubuntu keyserver, Arch Linux keyring and [my website](https://alex19ep.me/public/key.gpg).

**Comment by bomzhellino** [comment]
Здравствуйте. Пытаюсь  сделать словарь расшифровки сокращений. Почему-то если в сокращении есть гласная буква, то замена происходит корректно. Если же только согласные буквы, то замены нет.
Например:
Ит=итерация — читает правильно
Трн=транзакция — читает "тээрэн"
См=смотри — упорно читает "эсэм"
Смо=смотри — нормально читает "смотри".
Что не так? Может, есть какая-то опция в конфигурационном файле?
